tools/ja-rule/ja-rule.cpp

```diff
@@ -407,8 +709,20 @@ class InputHandler {
 int main(int argc, char **argv) {
   ola::AppInit(&argc, argv, "[ options ]", "Ja Rule Admin Tool");
 
+  auto_ptr<UID> target_uid(UID::FromString(FLAGS_target_uid));
+  if (!target_uid.get()) {
```

Comment:
Yep. Hence I wonder if it should just be a warning to STDOUT that doesn't affect the processing. "Are you sure you meant to do that?"